src/main/java/com/metamx/common/lifecycle/Lifecycle.java

```diff
@@ -243,7 +250,10 @@ public void addMaybeStartHandler(Handler handler, Stage stage) throws Exception
   public void start() throws Exception
   {
     synchronized (handlers) {
-      started.set(true);
+      if (!started.compareAndSet(false, true)) {
```

Comment:
Should this be an error? I think unlike stop, there's no good reason for start to be called multiple times